package.json

```diff
@@ -34,7 +34,10 @@
     "dist"
   ],
   "scripts": {
-    "build": "vite --mode production build",
+    "build": "npm run build:cleanup && npm run build:source && npm run build:version",
+    "build:cleanup": "tsc --build --clean",
+    "build:source": "tsc",
+    "build:version": "echo \"export const __PACKAGE_VERSION__ = '$npm_package_version'\" > dist/version.js",
```

Comment:
Why generate it and not use package.json in the module?

Reply:
Requires bumping Node version to at least 20.10 to support it without having to parse the json by our self (on runtime(!))

Reply:
Fixed the new pushed commit